app/posts/find-a-lost-trn/2022-05-06-performance-analysis.md

```diff
@@ -0,0 +1,67 @@
+---
+  title: Performance analysis in private beta
+  date: 2022-05-06
+---
+
+In private beta, we will collect data to report on the four mandatory KPIs, as well as additional data that will help us to understand our users and iterate the service. 
+
+## 1. Completion rate
+
+Initially, we will monitor the completion rate of the Find a lost TRN service by transaction, within the boundary of the digital service. In the future, we plan to monitor it by distinct user and include interactions with the TRA help desk.
+
+In the initial phase we will record:
+- Number of requests started
+- Number of TRNs found
+- Number of help desk tickets created
+- Number of requests abandoned without finishing, of which:
+    - Number of requests abandoned at each screen
+
+The completion rate is calculated as:
+
+(Number of TRNs found + number of help desk tickets created) / number of requests started
```

Comment:
Should we have a completion rate and a success rate? Where success is the number of TRNs found / number of requested started?